docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+case.
+
+  In situations when the general population is represented in the denominator
+  rather than the at risk population...
+
+
+    If the prevalence of a condition is *small*, ``1 - prevalence`` ~ ``1``.
+    In these cases, incidence calculated as the number of new cases per
+    person-years in the *entire* population will be *approximately* equal to
+    the number of new cases per person-years in the *at-risk* population.
+    Therefore, the approximation will be fairly accurate and likely not have a
+    large impact on the model transition rates.
+
+    If the prevalence of a condition is *large*, ``(1 - prevalence)`` < ``1``.
+    In these cases, the approximation will be more inaccurate and may bias the
+    model transition rates.
+
+  Therefore, it is important to understand how incidence data sources used for
+  cause models are measured and whether the population in the denominator
+  represents the at risk population or the general population. If the
+  population in the denominator represents the general population, the impact
+  on the model and potential solutions to limit bias should be considered.
+
+    A potential solution may be to represent the transition rate with the
+    following:
+
+    ``incidence rate`` * ``population size`` / ``(1 - prevalence)``
+
+Further, it is important to consider that cause models are *state*-specific
+and not necessarily *disease*-specific. What does this mean?
+
+  Consider a cause model in which an individual can transition from a
+  susceptible state to a mild disease state OR from a susceptible state to a
+  severe disease state.
+
+  In this case, the incidence rate for overall disease (mild and severe) does
+  not help us estimate the transition rates from susceptible to mild disease
+  or to severe disease. In these cases, incidence rates specific to mild and
+  severe disease are needed to inform the specific transitions present in the
+  cause model.
+
+Lastly, see the section on `hazard rates`_ in non-standard data sources below
+to determine when hazard rates may be preferable to annual incidence rates as
+a data source for cause model transition rates.
+
+Remission
 ++++++++++
 
-Cause-specific mortality
+Remission rates are defined as the **number of newly recovered cases from a
+condition that occur per person-year of the population with the condition.**
+
+  For example, the remission rate of type II diabetes in the United States is
+  2.4 per 10,000 person-years.
+
+  This suggests that if we followed 10,000 individuals with type II diabetes
+  for one year each, we would expect to see 2.4 individuals recover from type
+  II diabetes.
+
+Remission rates can be used to estimate cause model transition rates and
+represent the probability that a simulant in an infected (with condition)
+state will transition to a non-infected (without condition) state.
+
+  For example, with a time step of one year, the probability that a simulant
```

Comment:
Maybe actually use an infectious disease as an example.  The remission rate for diarrheal diseases is like 70.  And that kind of extremely high rate is important to talk about.